src/clj_puppetdb/http.clj

```diff
@@ -29,9 +30,9 @@
 
 (defn- make-client-common
   [^String host opts]
-  (let [opts (assoc opts :as :stream)
-        info (select-keys opts connection-relevant-opts)
-        info (assoc info :host host)]
+  (let [conn-opts  (select-keys opts connection-relevant-opts)
```

Comment:
eagle eye spot! will fix. i think we should probably get cljfmt applied to this repo, but probably not this release! :)